environment-r.yml

```diff
@@ -1,11 +0,0 @@
-name: R
-channels:
-  - defaults
-  - r
-dependencies:
-  - psutil
-  - nb_conda_kernels
-  - jupyter
-  - r-essentials
```

Comment:
We no longer need a separate env now that the packages are consistent with `conda-forge` versions. However, we still do not have `r-essentials`. That is a collection of the basic `R` packages an average users expects. I will work on that soon.